PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -83,6 +83,10 @@ extension DefaultControlsViewController {
         var animationsEnabled: Bool
         var contentFullScreenAction: Command
         
+        var isBrandedContentHidden: Bool
+        var brandedContentTitle: String
+        var showBrandedContentWebSite: Command
```

Comment:
Maybe `brandedContentCommand`? You don't know in controls module what command will arrive for that button.

Reply:
👍